packages/sommelier.rb

```diff
@@ -250,7 +232,7 @@ def self.build
           # some of these checks in an env.d file since we don't need
           # them run every time a shell is opened.
           # Get a list of all available DRM render nodes.
-          DRM_DEVICES_LIST=( /sys/class/drm/renderD* )
+          DRM_DEVICES_LIST=$(ls /dev/dri/renderD*)
```

Comment:
The previous one is better? (and this change breaks line 251)

Reply:
The previous check doesn't work and this does. :)